tsl/src/nodes/decompress_chunk/decompress_chunk.c

```diff
@@ -973,8 +973,10 @@ create_compressed_scan_paths(PlannerInfo *root, RelOptInfo *compressed_rel, int
 	Path *compressed_path;
 
 	/* clamp total_table_pages to 10 pages since this is the
-	 * minimum estimate for number of pages */
-	root->total_table_pages = Max(compressed_rel->pages, 10);
+	 * minimum estimate for number of pages.
+	 * Do not over-write any existing estimates
```

Comment:
This comment is misleading, since the actual code overwrites the existing value with added number of pages. `do not over-write` will do nothing if the value exists. Can you clarify the comment or fix the code?

Reply:
modified comment.